icomoon/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: "icomoon";
+  src: url("fonts/icomoon.woff?adhf8k") format("woff");
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"],
+[class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: "icomoon" !important;
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-ChevronDown:before {
```

Comment:
Please do not use CamelCase. Could you please transform it to `.icon-chevron-down`?